homework/tests/HomeworkTests.spec.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
+
+data.forEach(version => {
+
+
+test.describe(version + ': Add', () => {
```

Comment:
for readability, this line could also have indentation, to make it clear that it is on the same level with other describe blocks and within foreach.